### ISSUE TITLE
[Bug Report]: Wii Classic Controller Not Fully Working

### ISSUE THREAD

**Issue by Josh98VFW** [ndeadly/MissionControl#547]
### Switch Firmware Version

15.0.1 (Latest)

### Atmosphère Version

1.4.0 (Latest)

### Mission Control Version

0.8.0 (Latest)

### Boot Method

Fusée

### Issue Description

WII Classic Controller Buttons R and ZR not working.
R is Always pressed.
Controller fully working on Wii and Dolphin.

**Comment by Josh98VFW** [comment]
Attaching Wii Motion Plus resolves the problem

**Comment by ndeadly** [comment]
Confirmed that this is an issue on the current version. It was introduced in version 0.7.1 when I added a mapping for `L3`/`R3 ` buttons by pressling `1`/`2` on the wiimote, without the ability to test it myself. Turns out I overlooked a couple of things. 

This should now be working as intended.
[MissionControl-0.8.0-develop-de306be.zip](https://github.com/ndeadly/MissionControl/files/10211936/MissionControl-0.8.0-develop-de306be.zip)
